Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -123,5 +237,43 @@ function rotateTable(obj, table, axisString, value)
                 table.setRowAtIndex(iRow,rowVec_rotated)
             end
         end
+        function outputPath = generateOutputPath(obj,path, ext)
+            % Function to generate an output path from no, partial, or fully
+            % defined user path. 
+            
+            % Validate the output filename
+            if size(path,2) > 1
+                % More than 1 input to the method, only takes one (filepath)
+                error([ num2str(size(path,2)) ' inputs, expecting zero or one'])
+            end
+        
+            if isempty(path)
+               % No file path has been input, so use the path and name from
+               % the c3d file. 
+               filepath = obj.getPath();
+               name = obj.getName();
+            else
+            
+                if ~ischar(path{1})
+                   % Path must be a string  
+                   error('Input must be a sting of characters')
+                end
+
+                if isempty(strfind(path{1}, ext))
+                   % Path must have an input path
```

Comment:
You could remove this comment; it's redundant with the message in the error. Also, the message in the error is clearer than this comment :)